### PR TITLE
Mejora de selectores

### PR DESCRIPTION
**Pull request by MCLRDRI15**

### CODE REVIEW

File: cypress/page/address-step.page.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ class AddressStepPage {
   private AddressCheckout: string;
 
   constructor() {
-    this.AddressCheckout = "p.cart_navigation button";
+    this.AddressCheckout = "p.cart_navigation button[name='processAddress']";
```
Comment (holgiosalos) [inline cypress/page/address-step.page.ts]: probably, just with `button[name='processAddress']` or `[name='processAddress']` should be enough

File: cypress/page/shipping-step.page.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ class ShippingStepPage {
   private Checkout: string;
 
   constructor() {
-    this.AgreeTerms = "div.checker input";
+    this.AgreeTerms = "div.checker input[name='cgv']";
```
Comment (holgiosalos) [inline cypress/page/shipping-step.page.ts]: this element has an id, you can use it instead of the name.

File: cypress/page/shipping-step.page.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@ class ShippingStepPage {
...
     this.Checkout = "p.cart_navigation button[name='processCarrier']";
```
Comment (holgiosalos) [inline cypress/page/shipping-step.page.ts]: probably, just with button[name='processCarrier'] or [name='processCarrier'] should be enough

**Review by holgiosalos — COMMENTED** [review]
I have the feeling that you can improve the selectors on these page objects:
- login.page
- payment-step.page
- products-list.page
- shopping-cart.page

File: cypress/page/address-step.page.ts (hunk 1 above)
Comment (MCLRDRI15) [reply cypress/page/address-step.page.ts]: Thanks Giovanny, its done !!

File: cypress/page/shipping-step.page.ts (hunk 2 above)
Comment (MCLRDRI15) [reply cypress/page/shipping-step.page.ts]: 👍🏼

File: cypress/page/shipping-step.page.ts (hunk 3 above)
Comment (MCLRDRI15) [reply cypress/page/shipping-step.page.ts]: sure !!

File: cypress/page/products-list.page.ts [hunk 4]
```diff
@@ -5,7 +5,7 @@ class ProductListPage {
   private CheckoutButton: string;
 
   constructor() {
-    this.CartSelect = "#center_column a.button.ajax_add_to_cart_button.btn.btn-default";
+    this.CartSelect = ".ajax_add_to_cart_button > span";
     this.ModalView = "#layer_cart";
     this.CheckoutButton = ".clearfix div.button-container > a";
```
Comment (holgiosalos) [inline cypress/page/products-list.page.ts]: `a[title="Proceed to checkout"]` should work better for this

image

Comment (MCLRDRI15) [reply cypress/page/products-list.page.ts]: Sure Giovanny, thanks for all the support and reviews !!!

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**